java-snapshot-testing-plugin-jackson/src/main/java/au/com/origin/snapshots/jackson/serializers/v1/DeterministicJacksonSnapshotSerializer.java

```diff
@@ -16,6 +16,7 @@ public class DeterministicJacksonSnapshotSerializer extends JacksonSnapshotSeria
 
   @Override
   public void configure(ObjectMapper objectMapper) {
+    super.configure(objectMapper);
```

Comment:
This could potentially break existing implementations that are already subclassing without calling `super`.

What about a method that allows you to turn it off in subclasses?

JacksonSnapsohtSerializer
```
protected boolean shouldFindAndRegisterModules() {
    return true;
}
```

Reply:
You're right. Your suggestion is also fine for me.